templates/types/streaming/nextjs/app/api/data/[path]/route.ts

```diff
@@ -21,7 +21,11 @@ export async function GET(
   }
 
   try {
-    const filePath = path.join(process.cwd(), "data", slug);
+    let retrievedFolder = "data"; // default folder
```

Comment:
can we set the route to /api/[path]/route.ts and filter out requests that are not data nor tool-output?

Reply:
or maybe we can use the /api/files/[path]/route.ts endpoint instead (and [path] can contain data or tool-output? so we can easily extend it for other files?